packages/avatar/src/Avatar.js

```diff
@@ -67,22 +67,41 @@ function Image({ image, name, onError }) {
 // eslint-disable-next-line react/prop-types
 function Initials({ name }) {
   return (
-    <div className="hig__avatarV2__initials" aria-hidden="true">
+    <span className="hig__avatarV2__initials" aria-hidden="true">
       {initialsFromName(name)}
-    </div>
+    </span>
   );
 }
 
+const modifiersBySize = {
+  [sizes.SMALL_16]: "hig__avatarV2--size--small",
+  [sizes.MEDIUM_24]: "hig__avatarV2--size--medium",
+  [sizes.MEDIUM_32]: "hig__avatarV2--size--medium-32",
+  [sizes.LARGE_36]: "hig__avatarV2--size--large",
+  [sizes.LARGE_48]: "hig__avatarV2--size--large-48",
+  [sizes.XLARGE_64]: "hig__avatarV2--size--extralarge"
+};
+
+const modifiersByBackgroundId = {
+  "1": "hig__avatarV2__background--turquoise",
+  "2": "hig__avatarV2__background--purple",
+  "3": "hig__avatarV2__background--pink",
+  "4": "hig__avatarV2__background--salmon",
+  "5": "hig__avatarV2__background--blue",
+  "6": "hig__avatarV2__background--green",
+  "7": "hig__avatarV2__background--turquoise",
```

Comment:
`--turquoise` is in here twice.

Reply:
Yeah, it was already written that way in CSS. This removes redundant styles.
https://github.com/Autodesk/hig/blob/development/packages/avatar/src/avatar.scss#L51-L57
The `getBackgroundId` function needs to be rewritten to remove the duplication.

Reply:
ah got it. saw the redundancy removed in the css didn't know it was intentional to leave it here.